.gitignore

```diff
@@ -13,4 +13,5 @@
 *.a
 mkmf.log
 
-spec/vcr_cassettes/drivers.yml
\ No newline at end of file
+spec/vcr_cassettes/drivers.yml
```

Comment:
It was supposed to prevent other people looking at our data. At this point though, since we only use development seeded data, I will get rid of this line.